### ISSUE TITLE
[Suggest] Add a set-cookie middleware

### ISSUE THREAD

**Issue by FerroO2000** [middyjs/middy#418]
It would be nice to have an "out of the box" middleware for setting the set-cookie header

**Comment by lmammino** [comment]
Hello @FerroO2000! Thanks a lot for the suggestion.

Do you have some examples in mind for such a middleware?

**Comment by FerroO2000** [comment]
Sorry for the delay (I forgot it). At the end I have wrote a little and stupid middleware which handles (only for my needs) cors, cookies, http-errors and success

```javascript
module.exports = () => {
  return {
    after: (handler, next) => {
      const headers = {
        'Access-Control-Allow-Origin': 'http://localhost:3000',
        'Access-Control-Allow-Credentials': true
      };
      if (handler.response.cookie) {
        headers['Set-Cookie'] = handler.response.cookie;
      }
      handler.response = {
        statusCode: handler.response.statusCode || 200,
        headers,
        body: JSON.stringify({
          message: handler.response.message || 'Success',
          data: handler.response.data
        })
      };

      next();
    },
    onError: (handler, next) => {
      handler.response = {
        statusCode: handler.error.statusCode || 500,
        body: JSON.stringify({
          message: handler.error.message || 'Error'
        })
      };
      next();
    }
  };
};
```
